add info about how data comes from Clearbit (#4040)
segmentio/segment-docs: src/connections/destinations/catalog/clearbit-reveal/index.md

To verify that the destination has been set up correctly, send a page event **that includes an IP address**, check the Debugger section of your Segment Source. Assuming everything is as it should be, you should start seeing Clearbit Reveal data populate in an `identify` event – click on the specific event you're interested in to see Clearbit Reveal traits. These traits will now be available to other Segment destinations in your account. Notice that all Clearbit Reveal traits are prefixed with `reveal_` to ensure they don't conflict with existing traits. Clearbit will also send a `track` event for 'enrichment_provider'.

> When you make requests to Clearbit, they send events with Clearbit data back to your Segment source server-side using Segment's analytics-ruby library. If you see unexpected traffict from analytics-ruby in your Debugger, not to worry! Those are just your Clearbit events.

Reviewer: ```suggestion
When you make requests to Clearbit, Clearbit send events with its own data back to your Segment source server-side using Segment's analytics-ruby library. If you see unexpected traffic from analytics-ruby in your Debugger, that traffic represents the events that Clearbit sends back.
```

- Typo
- remove exclamation point
- Reword for clarity
